src/Resources/contao/library/Contao/DcaLoader.php

```diff
@@ -85,7 +85,10 @@ public function load($blnNoCache=false)
 					include $file;
 				}
 			}
-			catch (\InvalidArgumentException $e) {}
+			catch (\InvalidArgumentException $e)
+			{
+				\System::getContainer()->get('monolog.logger.contao')->notice($e->getMessage(), ['exception' => $e]);
```

Comment:
Both wrong IMHO.

> Well we can't change that for BC reasons, extending from `\InvalidArgumentException` is BC.

The `PaletteNotFoundException` class is only added in this PR, therefore there cannot be a BC break.

> And I think specifying an invalid palette is an `InvalidArgumentException` ;-)

Not necessarily. An invalid argument is something that is passed at runtime. A configuration error in a file (in this case a DCA file) is a logic error.